src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -82,12 +85,9 @@ public void AnalyzingTarget(IAnalysisContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            if (Verbose)
-            {
-                WriteLineToConsole(string.Format(CultureInfo.CurrentCulture,
+            WriteLineToConsole(string.Format(CultureInfo.CurrentCulture,
                     SdkResources.MSG001_AnalyzingTarget,
                         context.TargetUri.GetFileName()));
-            }
         }
 
         public void Log(ReportingDescriptor rule, Result result)
```

Comment:
so this call should *not* send results to the console when --quiet is passed. other loggers *will* persist these results to, say, a log file.

Reply:
Done.

---
In reply to: [559738602](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559738602) [](ancestors = 559738602)